src/installation/guides/fde.md

```diff
@@ -242,11 +238,15 @@ Change the permissions to protect generated the key.
 # chmod -R g-rwx,o-rwx /boot
 ```
 
-This keyfile also needs to be added to `/etc/crypttab`. Again, this will be
-`/dev/sda2` on EFI systems.
+This keyfile also needs to be added to `/etc/crypttab`. Again, this would be
+`/dev/vda1` on legacy BIOS systems.
 
 ```
-voidvm   /dev/sda1   /boot/volume.key   luks
+# blkid -o value -s UUID /dev/vda2
+```
+
+```
+voidvm   UUID=ADD-UUID-FROM-BLKID-COMMAND-HERE   /boot/volume.key   luks
```

Comment:
Refer to the `LUKSPART` variable to avoid repeatedly noting the distinction between BIOS and UEFI.